scss/index.scss

```diff
@@ -44,6 +44,11 @@ $highlight-ypos: 0.07em;
 
   display: flex;
   justify-content: center;
+
+  .applications-closed-message {
+    display: none;
+    color: #fdcb6e;
+  }
```

Comment:
We should consider just making the text disappear once the application closes.

Also, for reference, this is how it looks like right now: 
![image](https://user-images.githubusercontent.com/7832610/91504523-00d4bd00-e893-11ea-9d79-7b0f8be96703.png)

Reply:
If we're going to keep the Apply buttons on there or convert them into RSVP, I feel like having some messaging on the website that shows that applications are closed might be a good idea? Could reduce confusion / avoid any clarifying emails people might wanna send

Reply:
hmm.. that's a good point but maybe we should keep the naming similar to not confuse people (that its a new page). Maybe we could rename the button to RSVP but the nav should remain

Reply:
What do you mean by keep the naming similar? For the buttons? I think there's going to be some downtime between when applications close and when people can start RSVP-ing, so we might need to take a decision about what to do during that period